views/histogram.py

```diff
@@ -8,6 +8,8 @@
 from components.config import histogram_config
 from utils import earthquake_data
 from app import app
+from views.quake_map import get_datetime_from_str
```

Comment:
This function is actually in module `utils.dateutils` so it might be good form to import from there